presto-tests/src/test/java/com/facebook/presto/execution/TestQueues.java

```diff
@@ -45,7 +45,7 @@ public void testSqlQueryQueueManager()
         testBasic(false);
     }
 
-    @Test(timeOut = 240_000)
+    @Test(timeOut = 240_000, invocationCount = 100)
```

Comment:
Why run 100 times? Is it expected that some runs will fail?

Also, is the 240 second timeout appropriate? With 100 runs, that means we could wait nearly 7 hours.